### DISCUSSION TITLE
Elastic agent installation on slow servers

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#12823] [2.4] [answered]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello!

While installing Elastic Agent on some Win servers and PC's we faced up with a problem "exit status 1" because of [3 minutes timeout](https://github.com/Security-Onion-Solutions/securityonion/discussions/12515).
We have some really slow PC's that can generate this error, but we have to install Agent on them. How to regenerate installer without 3 minutes limit or with extended limit for 10 minutes?

Thanks a lot!

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by Sahale** [comment]
Any advice?

**Reply by defensivedepth** [reply]
We have an issue open to lengthen this timeout and make it configurable.

**Reply by Sahale** [reply]
Didn't see it, only found one about timeout error message.
Ok, will be waiting! Hope it'll be released soon.
Thanks!

**Comment by Sagatjkeee** [comment]
Hello!
We have same ploblem.
Can we expect a solution to this problem in the upcoming releases?
Or is there information about when we can approximately expect this decision?

**Comment by defensivedepth** [comment, marked as answer]
@Sahale @Sagatjkeee 

2.4.70 was released yesterday and it changes the default timeout to 5 minutes, as well as adds a command line parameter `timeout` that can be used to override the default.

**Reply by Sahale** [reply]
Tried to regenerate agents and use parameter, but got error:
```
> so-elastic-agent_windows_amd64.exe -timeout 10
Usage of C:\Blah\so-elastic-agent_windows_amd64.exe:
  -fleet string
        Override default Fleet Host
  -token string
        Override default Enrollment Token
```
How should I set parameter? Thanks!

**Reply by defensivedepth** [reply]
`-timeout=10m` should work
